index.json

```diff
@@ -117,6 +121,7 @@
     "sm": "2px",
     "md": "4px",
     "lg": "8px",
+    "xl": "16px",
```

Comment:
what about `yg` for Yuge

Reply:
As soon as macOS adds 32px border-radii 😜